### PR TITLE
Make artifactory default docker registry.

### PR DESCRIPTION
**Pull request by paul-ktech**
And drop using ecr.

### CODE REVIEW

File: charts/bandstand-web-service/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: bandstand-web-service
-version: 1.6.12
+version: 1.7.0
```
Comment (gavinclarkeuk) [inline charts/bandstand-web-service/Chart.yaml:3]: Again 2.0.0

File: charts/bandstand-cron-job/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: bandstand-cron-job
-version: 1.6.0
+version: 1.7.0
```
[the remark this answers is not in the document]
Comment (paul-ktech) [reply charts/bandstand-cron-job/Chart.yaml:3]: Backstage templates reference versions ~x.y.z so they should be only updating on patch changes. I was going to updating their minor version next, plus their ktech docker orb version to ensure that images are pushed to artifactory. new template-generated projects should work fine then with existing unaffected.

I'll publish instructions for upgrading existing projects to use latest orb + charts.
.

[the remark this answers is not in the document]
Comment (gavinclarkeuk) [reply charts/bandstand-cron-job/Chart.yaml:3]: I understand that - just thought we were following semver conventions, whatevs

[the remark this answers is not in the document]
Comment (gavinclarkeuk) [reply charts/bandstand-cron-job/Chart.yaml:3]: Also I thought we were going to do an orb v3 which dropped ECR support before rolling out more widely/to the templates

**Review by gavinclarkeuk — APPROVED** [review]

**Pull request merged**